api/jobset/v1alpha2/jobset_types.go

```diff
@@ -24,6 +24,7 @@ const (
 	ReplicatedJobReplicas string = "jobset.sigs.k8s.io/replicatedjob-replicas"
 	ReplicatedJobNameKey  string = "jobset.sigs.k8s.io/replicatedjob-name"
 	JobIndexKey           string = "jobset.sigs.k8s.io/job-index"
+	NamespacedJobNameKey  string = "jobset.sigs.k8s.io/namespaced-job-name"
```

Comment:
we need to use this key instead of JobNameKey in [`setExclusiveAffinities`](https://github.com/kubernetes-sigs/jobset/blob/main/pkg/controllers/jobset_controller.go#L608)

Reply:
right, of course - done!